src/xcode/ENA/ENA/Source/Models/ClientMetaData/ClientMetadata.swift

```diff
@@ -0,0 +1,53 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+struct ClientMetadata: Codable {
+	var cwaVersion: Version?
+	var iosVersion: Version
+	var eTag: String?
+
+	enum CodingKeys: String, CodingKey {
+		case cwaVersion
+		case iosVersion
+		case eTag
+	}
+
+	init(etag: String?) {
+
+		let iosVersion = ProcessInfo().operatingSystemVersion
+		self.iosVersion = Version(
+			major: iosVersion.majorVersion,
+			minor: iosVersion.minorVersion,
+			patch: iosVersion.patchVersion
+		)
+		
+		guard let majorAppVerson = Int(AppStrings.Home.appInformationVersion),
+			  let minorAppVerson = Int(Bundle.main.appVersion),
+			  let patchAppVersion = Int(Bundle.main.appBuildNumber) else {
```

Comment:
`AppStrings.Home.appInformationVersionI` is, i think, only a label printing "Version". This won't work.
Also for 'Bundle.main.appVersion' and `Bundle.main.appBuildNumber` I am not sure if this will work. These are String, which could include everything then a number.
Perhaps write a unit test to see if it really works (see comment below)

Reply:
yes you are right, should have tested that, thanks!